### ISSUE TITLE
Eager to get the code, too. Curious about the detail structure about the Scene Flow Prediction Model.

### ISSUE THREAD

**Issue by CTouch** [kaichen-z/DynPoint#2]
I try to construct my own network according to the figure 7 in your paper, but it does not work well.

Perhaps the preprocessing part is different. I would like to ask what depth and optical flow estimation algorithms you use.

**Comment by kaichen-z** [comment]
Thank you for your patience. I just released the code for the depth estimation and scene flow estimation part of our paper.

**Comment by CTouch** [comment]
Thanks for your work and reply!
I notice that the dataset examples(DAVIS) provided have a accurate mask for motion segmentation. But in Nvidia dataset there is no accurate mask, and I have difficult to train the scene flow in it. The scene flow model likely generate small scene flow(almost zero). Could you please release one example scene from Nvidia dataset?
Looking forward to your reply!

**Comment by kaichen-z** [comment]
Certainly, I'll share the motion mask generated by our processing of the Nvidia Dataset. Additionally, please be aware that the pose estimation in the Nvidia dataset relies on colmap, which may not be entirely accurate. We recommend using our regularization masks to refine the poses for improved accuracy.
